docs/pages/access-controls/teleport-policy/getting-started-policy.mdx

```diff
@@ -13,15 +13,16 @@ It can help you answer questions like:
 
 ## Getting started with Teleport Policy
 
-Access Graph is a feature of the Teleport Policy product that is only 
-available to Teleport Enterprise customers.
+Teleport Policy is a separately licensed product and is available to Teleport Enterprise customers.
+Access Graph is a major capability of Teleport Policy that visually shows the relationships of 
+policies of users, groups, and computing resources.
 
-After logging into the Teleport UI, go to the Management tab. If enabled, Access Graph options can be found
-under the Permission Management section.
+After logging into the Teleport UI, go to the Management tab. If enabled, Teleport Policy’s Access Graph options 
+can be found under the Permission Management section.
 
 <Admonition type="note">
-Note: For managed Teleport Enterprise customers, Access Graph is enabled by default.
-If you are a self-hosted Teleport customer, you will need to set up [Access Graph](../access-graph/self-hosted.mdx) and ensure you have an updated 
+Note: For managed Teleport Enterprise customers, Teleport Policy is enabled by default.
```

Comment:
I understand that we're trying to avoid saying "Teleport Cloud" here, but I feel like we're making things more complicated. Do we think people will know that "managed customers" means cloud?

Reply:
It is rather long. We use 'managed Teleport Enterprise' in a few other places. Does 'managed Enterprise customers' sound better?